README.md

```diff
@@ -89,4 +89,4 @@ Homebrew is a member of the [Software Freedom Conservancy](https://sfconservancy
 
 [![Software Freedom Conservancy](https://sfconservancy.org/img/conservancy_64x64.png)](https://sfconservancy.org)
 
-Homebrew is generously supported by [Zeno R.R. Davatz](https://github.com/zdavatz), [Randy Reddig](https://github.com/ydnar) and 262 other users via [GitHub Sponsors](https://github.com/sponsors/Homebrew).
+Homebrew is generously supported by [Randy Reddig](https://github.com/ydnar) and 271 users via [GitHub Sponsors](https://github.com/sponsors/Homebrew).
```

Comment:
Thoughts on reducing the number to avoid this changing too often?

Reply:
Could be changed to “almost 300 users” (and then “over 300 users” when that milestone is passed). Alternatively, “over 250 users” and keep incrementing every 50.

Reply:
> Thoughts on reducing the number to avoid this changing too often?

Perhaps @dawidd6 could look into https://github.com/tj/sponsors-api and https://developer.github.com/v4/object/sponsorship/ (or https://docs.github.com/en/graphql/reference/mutations#sponsorship)